src/__test__/homepage/homepage.test.js

```diff
@@ -0,0 +1,66 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import App from '../../App';
+
+describe('Homepage', () => {
+  it('should render the homepage', async () => {
+    const json = {
+      json: () => Promise.resolve([
+        {
```

Comment:
[ Optional Changes ]

- [ ] The console is showing warnings on your test for this component relating to prop types. Please check on that 👍🏽